src/utils/TriangleUtilities.js

```diff
@@ -86,11 +86,12 @@ export function getTriangleHitPointInfo( point, geometry, triangleIndex, target
 	// find the associated material index
 	let materialIndex = 0;
 	const groups = geometry.groups;
+	const firstVertexIndex = triangleIndex * 3;
 	for ( let i = 0, l = groups.length; i < l; i ++ ) {
 
 		const group = groups[ i ];
 		const { offset, count } = group;
-		if ( triangleIndex >= count && triangleIndex < offset + count ) {
+		if ( firstVertexIndex >= count && firstVertexIndex < offset + count ) {
```

Comment:
Also the check with the beginning of the group should be ```start```, not ```count```
So this line should be:
```if ( firstVertexIndex >= start && firstVertexIndex < start + count ) {```